src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -454,6 +470,250 @@ components:
         - actionedBy
         - timestamp
         - correlationId
+    CiagInductionResponse:
```

Comment:
> I've included Ciag in the name of both so that we can easily add our own versions in the future (though I appreciate I haven't done this throughout with the other related classes, so this idea is somewhat flawed!).

LOL - I was just going to ask about that, but you've answered it 😁 No problem with this approach, and I like your thinking about futures etc 👍